test/client/DummyPackageManagementServiceClient.ts

```diff
@@ -34,26 +34,35 @@ export class DummyPackageManagementServiceClient implements IPackageManagementSe
     }
 
     listKnownPackages(
-        request: ListKnownPackagesRequest, 
-        callback: (error: ServiceError, 
+        request: ListKnownPackagesRequest,
+        callback: (error: ServiceError,
                    response: ListKnownPackagesResponse) => void
     ): ClientUnaryCall;
 
     listKnownPackages(
-        request: ListKnownPackagesRequest, 
-        metadata: Metadata, 
+        request: ListKnownPackagesRequest,
+        metadata: Metadata,
+        callback: (
+            error: ServiceError,
+            response: ListKnownPackagesResponse) => void
+    ): ClientUnaryCall;
+
+    listKnownPackages(
```

Comment:
Haven’t tracked down what exactly changed to require the extra type annotation for the version with options now but it makes sense to have it and we already did have it for other services.